CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.0.0-beta.0
+
+- Convert to TypeScript
+- Change default export to named export `FullStory` 
+- Update `@fullstory/snippet` dependency to use version `2.0.0-beta.2`
+
```

Comment:
The big one: adopts the V2 browser API - https://developer.fullstory.com/browser/v2/getting-started/